modules/store/src/store_module.ts

```diff
@@ -58,19 +59,28 @@ export class StoreFeatureModule implements OnDestroy {
   constructor(
     @Inject(STORE_FEATURES) private features: StoreFeature<any, any>[],
     @Inject(FEATURE_REDUCERS) private featureReducers: ActionReducerMap<any>[],
+    @Inject(FEATURE_NAME) private featureName: string,
```

Comment:
`@Inject(FEATURE_NAME) private featureNames: string,`

Reply:
This should be a `string[]` because its a multi-provider.